apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/synthea_pre_validation_debug.sql

```diff
@@ -0,0 +1,256 @@
+-- This SQL mimics the function SYNTHEA_LOAD_PRE_VALIDATE, but does so
+-- by breaking apart the large 'union' of values into individual named
+-- values so one can see when/why a Synthea pre-valiation may have failed.
```

Comment:
Good idea! I suppose we can manually run this if something fails, but I wonder if this shouldnt just get run if we fail so it prints out the reason? Unless this takes much longer or some reason we wouldnt want to.

Reply:
Well I suppose we could, although that becomes kinda heavy-weight (as written) since it performs the entire gamut of checks vs. the actual function which operates on a 'fail fast' model. The thought was to provide some visibility into the number of failures that could be encountered from the end_state.properties data, so it was intended as more of a debugging tool.